gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -22,6 +22,11 @@
 import logging
 from collections import namedtuple
 
+import os
+import tempfile
+
+from nucleus.io.python import vcf_reader as nucleus
+from nucleus.protos import variants_pb2 as nucleus_proto
```

Comment:
Please use `variants_pb2` here. The one above is fine as you like to distinguish between pyvcf and nucleus parsers, but you should leave this one as is (e.g. it's easier for searching).